sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/CommunicationListener.java

```diff
@@ -0,0 +1,66 @@
+package com.azure.android.communication.chat.signaling;
+
+import com.microsoft.trouterclient.ITrouterConnectionInfo;
+import com.microsoft.trouterclient.ITrouterListener;
+import com.microsoft.trouterclient.ITrouterRequest;
+import com.microsoft.trouterclient.ITrouterResponse;
+
+import org.json.JSONObject;
+
+public class CommunicationListener implements ITrouterListener {
+
+    String chatEventId;
+
+    private RealTimeNotificationCallback listenerFromConsumer;
+
+    public CommunicationListener(String chatEventId, RealTimeNotificationCallback listener) {
+        this.chatEventId = chatEventId;
+        this.listenerFromConsumer = listener;
+    }
+
+    @Override
+    public void onTrouterConnected(String endpointUrl, ITrouterConnectionInfo connectionInfo) {
+        final String msg = "onTrouterConnected(): url=" + endpointUrl + ", newPublicUrl=" + Boolean.toString(connectionInfo.isNewEndpointUrl());
+        System.out.println(msg);
```

Comment:
I don't think we want to be printing directly to the system console. We should consider updating this to use the Azure Core logger or some other logging mechanism.

Reply:
Then same goes for all other places where we use `System.out.something`.